landingai/vision_pipeline.py

```diff
@@ -89,7 +95,7 @@ def from_array(cls, array: np.ndarray, is_BGR: bool = True) -> "FrameSet":
         array : np.ndarray
             Image
         is_BGR : bool, optional
-            Assume OpenCV's BGR color space? Defaults to True
+            Assume OpenCV's BGR channel ordering? Defaults to True
```

Comment:
If we want the SDK to be more friendly, maybe a better way is to handle RGB conversion at the source level: image file, video, camera etc. For simple cases like image file reading, leverage PIL is a better approach.

This parameter is a little problematic and I'd just remove it, since
1) it's hard to tell if a numpy array is BGR or RGB as the layout information is lost at this point. So the user may not even know it, especially for novice CV developers.
2) aside from BGR, and RGB, there are other common layouts like grayscale, rgba, etc. It's probably better to use an Enum with a dedicated function to support it instead of a flag.

(btw, the naming should be `is_bgr` instead (snake case), and the default value/behavior should probably be RGB)

Reply:
Note that the pipeline uses only PIL on its interfaces for the same reason that you mentioned (or at least I tried). In this case since this is an input/interface function we need it since the array may be coming from other sources

You have a good point about the other layouts... in fact we had an issue `overlay_bboxes` because it could not handle RGBA.... I have added a todo 

snake case comment:done